data/json/recipes/recipe_others.json

```diff
@@ -5707,15 +5708,15 @@
   "reversible": false,
   "autolearn": true,
   "qualities":[
-    {"id":"CHEM","level":2,"amount":1},
+    {"id":"CHEM","level":1,"amount":1},
```

Comment:
Haven't tried plasticizing milk IRL but I'm thinking it's not something I can make in a soup pot.  Cites?